django_cotton/tests/test_basic.py

```diff
@@ -160,7 +138,6 @@ def test_only_gives_isolated_context(self):
         with self.settings(ROOT_URLCONF=self.url_conf()):
             response = self.client.get("/only_view/")
             self.assertNotContains(response, "herebedragons")
-            self.assertNotContains(response, "only")
```

Comment:
@mattbha no, I just couldn't see why we're testing for 'only'. If we included an `{{ attrs }}` somewhere in the test template, it might make sense, to ensure it wasn't being considered a fall through boolean attribute.